spec/system/polls/polls_spec.rb

```diff
@@ -129,6 +129,19 @@
 
       expect(page).to have_link("Poll with results", href: results_poll_path(poll.slug))
     end
+
+    scenario "Shows SDG tags when feature is enabled" do
+      Setting["feature.sdg"] = true
+      Setting["sdg.process.polls"] = true
+
+      create(:poll, sdg_goals: [SDG::Goal[1]],
+                    sdg_targets: [SDG::Target["1.1"]])
+
+      visit polls_path
+
+      expect(page).to have_selector(".sdg-goal-tag-list")
+      expect(page).to have_selector(".sdg-target-tag-list")
```

Comment:
How about testing it from the user's point of view?

```suggestion
      expect(page).to have_selector "img[alt='1. No Poverty']"
      expect(page).to have_content "target 1.1"
```